tools/install_prereqs.sh

```diff
@@ -60,6 +60,7 @@ else
     libgfortran5 \
     hipblas-dev \
     hipblaslt-dev \
+    hipcc \
```

Comment:
There should probably be a comment that this is a workaround for broken hip-dev package.

Reply:
Added above